src/routes/api/locale.ts

```diff
@@ -0,0 +1,27 @@
+import type { RequestEvent } from '@sveltejs/kit';
+
+import { getLocale } from '$lib/utilities/i18n.utilities';
+
+// Supported locales
+import ca_en from '$locales/ca/en/index.json';
+import ca_fr from '$locales/ca/fr/index.json';
+import us_en from '$locales/us/en/index.json';
+import us_es from '$locales/us/es/index.json';
+
+const locales = {
+  ca_en,
+  ca_fr,
+  us_en,
+  us_es,
+}
+
+export async function get(event: RequestEvent) {
```

Comment:
This is just an API endpoint that can be used for fetching localized content, but it's not really used right now.

Reply:
Is this exclusively for E2E tests, or did you mean it is used by E2E as well as the app as a whole?